.pre-commit-config.yaml

```diff
@@ -15,27 +15,27 @@ repos:
 
   # Sort imports.
   - repo: https://github.com/pycqa/isort
-    rev: 5.9.3
+    rev: 5.10.1
```

Comment:
Are you planning to manually update `pre-commit` hooks?

Reply:
Well, there’s [autoupdate](https://pre-commit.com/#updating-hooks-automatically) but I guess we’ll have to run that every so often…

Reply:
I recommend to add it as an `Actions workflow` and schedule it to run and create a PR every month for example.

Reply:
As in… a [scheduled job](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onschedule)?

Reply:
> 

Yes, that would be great I think.

Reply:
@behnazh, should we open an issue for this?

Reply:
Something like the [pre-commit-autoupdate](https://github.com/marketplace/actions/pre-commit-autoupdate) action? Looks like heaps of options are available here.

Reply:
Yes, opening a new issue sounds good. I prefer not using a third-party action for the pre-commit autoupdate though because it's fairly simple.

Reply:
Issue https://github.com/jenstroeger/python-package-template/issues/63.